readthedocs/source/orb/operationcaching.md

```diff
@@ -0,0 +1,84 @@
+# Operation/Resolution Caching
+
+Orb node provides two unique caching features for resolving DIDs:
+1. resolve DIDs using unpublished operations
+2. resolve DIDs using published/unpublished operations from anchor origin
+
+## Operation/Resolution Caching Parameters
+
+The following section enumerates the startup parameters that have to be configured in order to enable 
+operation caching and resolution from anchor origin domain.
+
+### enable-unpublished-operation-store
+
+| Arg                                  | Env                                 | Caching |
```

Comment:
I think parameters should be documented in the parameters.md doc and you can link to them in this doc.

Reply:
Sure, I can use link to params. Detailed explanation has to be here though.